client/webserver/api.go

```diff
@@ -1129,13 +1129,7 @@ func (s *WebServer) actuallyLogin(w http.ResponseWriter, r *http.Request, login
 		}
 	}
 
-	writeJSON(w, struct {
-		OK    bool               `json:"ok"`
-		Notes []*db.Notification `json:"notes"`
-	}{
-		OK:    true,
-		Notes: loginResult.Notifications,
-	}, s.indent)
+	writeJSON(w, simpleAck(), s.indent)
```

Comment:
Although `(*Core).Login` no longer returns the notes, the `/api/login` http handler could retrieve the notifications after login and return that in the response, with the identical response structure.
Looking at the JS, it looks like it always does `/api/getnotes` after `/api/login`.  Is there a use case for having them separate?  Seems it could be made more efficient for the frontend.

Reply:
Good idea.